life_expectancy/cleaning.py

```diff
@@ -2,21 +2,35 @@
 import argparse
 import pandas as pd
 
-def clean_data(region: str) -> None:
-    """ Clean the data for a given region.
+def main(region: str) -> None:
+    """Main function."""
+    full_data = _load_data('life_expectancy/data')
+    region_data = _clean_data(full_data, region)
+    _save_data_csv('life_expectancy/data', region_data, region)
+
+def _clean_data(data: pd.DataFrame, region: str) -> pd.DataFrame:
```

Comment:
QUESTION: I see that you are using function names starting with an underscore. In general, how do you decide if to use an underscore or not in the function name?